src/Compilers/Test/Resources/Core/NetFX/Minimal/mincorlib.cs

```diff
@@ -2,8 +2,8 @@
 
 namespace System
 {
-	public class Object {}
-    public struct Byte {}
+    public class Object { }
```

Comment:
@tmat There is no blessed notion of minimal corlib, every dev has his own opinion on that matter or even on the question whether it should exist. If we want to have a discussion about that, it is outside of the scope of this PR.